assets/schema/ensemble_schema.json

```diff
@@ -4,24 +4,27 @@
   "title": "Ensemble",
   "type": "object",
   "properties": {
-    "Import": {
-
-    },
+    "Import": {},
     "ViewGroup": {
       "$ref": "#/$defs/Menu",
       "description": "Group multiple Views together and put them behind a menu."
     },
     "View": {
       "type": "object",
       "description": "This is your root View. It requires a body widget.",
-      "required": ["body"],
+      "required": [
```

Comment:
Do these formatting changes have any consequences @vusters ?

Reply:
should be ok. They are just json formatted differently.